src/main/java/project/bookstore/mapper/CartItemMapper.java

```diff
@@ -0,0 +1,20 @@
+package project.bookstore.mapper;
+
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import project.bookstore.config.MapperConfig;
+import project.bookstore.dto.cartitem.CartItemDto;
+import project.bookstore.dto.cartitem.CreateCartItemRequestDto;
+import project.bookstore.dto.cartitem.UpdateCartItemDto;
+import project.bookstore.model.CartItem;
+
+@Mapper(config = MapperConfig.class, uses = BookMapper.class)
+public interface CartItemMapper {
+    CartItem toModel(CreateCartItemRequestDto requestDto);
```

Comment:
Also add the method below to `BookMapper` class:

```
@Named("bookById")
    default Book bookById(Long id) {
        return Optional.ofNullable(id)
                .map(Book::new)
                .orElse(null);
    }
```

```suggestion
    @Mapping(target = "book", source = "bookId", qualifiedByName = "bookById")
    CartItem toModel(CreateCartItemRequestDto requestDto);
```